### ISSUE TITLE
Replace HTML tags tokens from translation

### ISSUE THREAD

**Issue by AlexandrTsvetkov** [i18next/jquery-i18next#44]
## 🚀 Feature Proposal

Implement [Trans](https://react.i18next.com/latest/trans-component#using-with-react-components) react component functionality for jquery

Let's assume that in translation I do have text like:
`"userMessagesUnread": "Hello <1>{{name}}</1>, you have {{count}} unread message."`

It would be nice if on client side code I can define by myself what kind of HTML tags I want to use for `<1></1>`
So, usage will be:
```
<p data-i18n="userMessagesUnread">
  Hello <strong>{{name}}</strong>, you have {{count}} unread message
</p>
```

## Motivation

Using separation of concerns principle we don't need to store HTML tags directly in translations

**Comment by jamuhl** [comment]
https://github.com/i18next/jquery-i18next#set-innerhtml-attributes does already what you ask for

**Comment by AlexandrTsvetkov** [comment]
@jamuhl Thanks for answer.
What do you think about adding additional information to docs with examples how to use it?

**Comment by jamuhl** [comment]
feel free to provide a PR improving the docs...(technically it's nothing else then setting innerHtml on an html element)

**Comment by AlexandrTsvetkov** [comment]
@jamuhl  Yes, sure. Wanted to ask one more thing. Maybe I am doing something wrong, but as an output I got this html code
```
<p data-i18n="[html]userMessagesUnread">
Hello &lt;1&gt;Name<!--1-->, you have {{count}} unread message.
</p>
```
Which looks like in the browser:
<img width="400" alt="image" src="https://github.com/i18next/jquery-i18next/assets/6397482/cf51109d-5cf8-4a2b-9b0a-889d342f18da">

Could you please advise me? Thanks!

**Comment by adrai** [comment]
Here you have an example: https://github.com/i18next/jquery-i18next/blob/9fe8b487c93e2779d52fa149d7ecb88e56442dad/example/simple/sample.html#L19

**Comment by AlexandrTsvetkov** [comment]
@adrai Thanks for providing link.
Unfortunately, in this example we use concrete tag `<strong>` directly in translation
What I wanted to achieve in to put in translation `<1>` tag as a placeholder and later define it in html code as I mentioned in the first post. Same as `<Trans>` component does from react library

`"userMessagesUnread": "Hello <1>{{name}}</1>, you have {{count}} unread message."`
```
<p data-i18n="userMessagesUnread">
  Hello <strong>{{name}}</strong>, you have {{count}} unread message
</p>
````

**Comment by adrai** [comment]
No, there is no Trans component for jquery-i18next

**Comment by AlexandrTsvetkov** [comment]
@adrai  Exactly, that's why this issue is the feature proposal :)

**Comment by adrai** [comment]
There's no plan for that. Feel free to contribute with a PR if you like to.
